src/software/firmware/test/test_pression.cpp

```diff
@@ -0,0 +1,33 @@
+/**
+ * @file test_pression.cpp
+ * @author Makers For Life
+ * @brief Unit tests for pression.cpp
+ * @version 0.1
+ * @date 2020-04-06
```

Comment:
Is it really relevant to have the version number and date here? I mean we have git and everyone will probably forget to update it when they edit the file...

Reply:
No, It's my auto generate Doxygen plugging that puts it automatically. Removed it